_posts/README.md

```diff
@@ -55,6 +86,8 @@ image: /assets/images/blog/encasement/encasement1.png
 
 description: "In 1986 a nuclear reactor known as Chernobyl released harmful radioactivity which spread over much of the western USSR and Europe. The core of this reactor remains a glowing, ineradicable mass of deadly radioactive lava in the middle of a large Exclusion Zone unfit for human habitation."
 
+excerpt: "In 1986 a nuclear reactor known as Chernobyl released harmful radioactivity which spread over much of the western USSR and Europe. The core of this reactor remains a glowing, ineradicable mass of deadly radioactive lava in the middle of a large Exclusion Zone unfit for human habitation."
+
```

Comment:
Worth mentioning in here somewhere that `description` and `excerpt` should be plain-text, no HTML or Markdown.